### ISSUE TITLE
[sklearn#34254] Path to moving from hosting the site scikit-learn/scikit-learn.github.io to cloudflare

### ISSUE THREAD

**Issue by godofecht** [flooooooooooow/flow#343]
**Mirrored from [scikit-learn#34254](https://github.com/scikit-learn/scikit-learn/issues/34254)**

Original title: Path to moving from hosting the site scikit-learn/scikit-learn.github.io to cloudflare
Original labels: Documentation, Build / CI, Hard
Created: 2026-06-11T15:50:58Z

---

This is the plan to get `scikit-learn.org` hosted on Cloudflare

### Requirements
1. `scikit-learn.org`'s website is on Cloudflare.
2. The versioned docs continue to live in https://github.com/scikit-learn/scikit-learn.github.io.
3. The `dev` website lives in Cloudflare and does not get pushed to https://github.com/scikit-learn/scikit-learn.github.io.

### Implementation path
1. Update https://github.com/scikit-learn/scikit-learn/blob/main/build_tools/circle/list_versions.py so the `zip` and `pdf` files point to Github. (A future extension is to upload `zip` and `pdf` to Cloudflare R2)
2. Manually deploy current website to Cloudflare.
3. Update https://github.com/scikit-learn/scikit-learn/blob/main/build_tools/circle/push_doc.sh to push to Cloudflare for `main` branch and stop pushing to `scikit-learn/scikit-learn.github.io`
4. Update https://github.com/scikit-learn/scikit-learn/blob/main/build_tools/circle/push_doc.sh or add a new github action in `scikit-learn/scikit-learn.github.io` to push to cloudflare when there is a release. (`1.9`, etc)
5. Update scikit-learn.org DNS to point to Cloudflare.

### Access to Cloudflare
Currently @ogrisel @thomasjpfan @adrinjalali @lesteve have access to the Cloudflare account

---

## Flow context

This issue is mirrored from scikit-learn to explore whether Flow encounters the same problem or can approach it differently. Add analysis below.

**Comment by godofecht** [comment]
Moved to https://github.com/godofecht/flow-scikit/issues/80 in the flow-scikit repository. Further library-specific triage and implementation will happen there.
